content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -116,9 +118,26 @@ function groupSitesByDate(sites) {
     }
     groupedSites.get(day).push(site);
   }
+  groupedSites.forEach((value, key) => {
+    const sessions = groupSitesBySession(value);
+    groupedSites.set(key, sessions);
+  });
   return groupedSites;
 }
 
+function groupSitesBySession(sites) {
```

Comment:
nit: `links` or `pages` not `sites`.

A site is `facebook.com`, a link is `https://facebook.com/profiles/me`